src/controller/RaceController.js

```diff
@@ -0,0 +1,26 @@
+import InputView from "../view/InputView.js";
+import Race from "../model/Race.js";
+import Winner from "../model/Winner.js";
+import OutputView from "../view/OutputView.js";
+
+const inputView = new InputView();
+const raceinstance=new Race();
+const winnerinstance =new Winner();
```

Comment:
```suggestion
const raceInstance=new Race();
const winnerInstance =new Winner();
```

camel case로 표현해주는 것도 좋을 거 같아요!